accessors/spanner/mocks.go

```diff
@@ -18,18 +18,25 @@ import (
 
 	spanneradmin "github.com/GoogleCloudPlatform/spanner-migration-tool/accessors/clients/spanner/admin"
 	spinstanceadmin "github.com/GoogleCloudPlatform/spanner-migration-tool/accessors/clients/spanner/instanceadmin"
+	"github.com/GoogleCloudPlatform/spanner-migration-tool/internal"
 )
 
 // Mock that implements the SpannerAccessor interface.
 // Pass in unit tests where SpannerAccessor is an input parameter.
 type SpannerAccessorMock struct {
-	GetDatabaseDialectMock          func(ctx context.Context, adminClient spanneradmin.AdminClient, dbURI string) (string, error)
-	CheckExistingDbMock             func(ctx context.Context, adminClient spanneradmin.AdminClient, dbURI string) (bool, error)
-	CreateEmptyDatabaseMock         func(ctx context.Context, adminClient spanneradmin.AdminClient, dbURI string) error
-	GetSpannerLeaderLocationMock    func(ctx context.Context, instanceClient spinstanceadmin.InstanceAdminClient, instanceURI string) (string, error)
-	CheckIfChangeStreamExistsMock   func(ctx context.Context, changeStreamName, dbURI string) (bool, error)
-	ValidateChangeStreamOptionsMock func(ctx context.Context, changeStreamName, dbURI string) error
-	CreateChangeStreamMock          func(ctx context.Context, adminClient spanneradmin.AdminClient, changeStreamName, dbURI string) error
+	GetDatabaseDialectMock          	func(ctx context.Context, adminClient spanneradmin.AdminClient, dbURI string) (string, error)
+	CheckExistingDbMock             	func(ctx context.Context, adminClient spanneradmin.AdminClient, dbURI string) (bool, error)
+	CreateEmptyDatabaseMock         	func(ctx context.Context, adminClient spanneradmin.AdminClient, dbURI string) error
+	GetSpannerLeaderLocationMock    	func(ctx context.Context, instanceClient spinstanceadmin.InstanceAdminClient, instanceURI string) (string, error)
+	CheckIfChangeStreamExistsMock   	func(ctx context.Context, changeStreamName, dbURI string) (bool, error)
+	ValidateChangeStreamOptionsMock 	func(ctx context.Context, changeStreamName, dbURI string) error
+	CreateChangeStreamMock          	func(ctx context.Context, adminClient spanneradmin.AdminClient, changeStreamName, dbURI string) error
+	CreateDatabaseMock					func(ctx context.Context, adminClient spanneradmin.AdminClient, dbURI string, conv *internal.Conv, driver string, migrationType string) error 
```

Comment:
nit: even here the formatting seems to be off, do you have gofmt installed?

Reply:
This is a github issue, the code looks fine on my IDE and will be fine when I merge it